ruby/src/api.cc

```diff
@@ -235,6 +237,7 @@ namespace leatherman { namespace ruby {
     string api::to_string(VALUE v) const
     {
         v = rb_funcall(v, rb_intern("to_s"), 0);
+        v = rb_str_encode(v, utf8_value("UTF-8"), 0, _nil);
```

Comment:
Should this be whatever locale is configured for the current program? i.e. query `leatherman::locale::get_locale()`.